Gaijinizer patch/Scripts.txt

```diff
@@ -3100,33 +3099,33 @@ What you are doing 'intervention' should have something to do with it.
  我々は、その時を心待ちにしております。
  =================================================="
 > CONTEXT: Scripts/テキスト出力/212:12
-"Subject: Congratulations
-Sender [Meira]
-
- 《Text》==========================================
- Dr. Kurosawa. Has invented another means of communication than the 'device'
- We will send you a message here.
-
-First of all, congratulations on 'completion of the story'.
- Whatever the outcome, he escaped that city
- The 'help' you can do in the world is over.
-But...I think you feel thin
- Apparently, it seems far from our hope of ending.
-
-If you are not convinced by this end
- If you are prepared to aim for a better end ...
-We are for you to see that world again
-There is a preparation.
-Of course, it is not 'the same world.'
-You will need to choose 'Start from scratch'.
-
-But you are different from the first time ...
-By leading one story to the end
- 'The girls' in my room will surely help you.
-Your experience and mind corruption will be taken over and helpful.
-Please be courageous and step into that world again.
- We look forward to that time.
- =================================================="#MTLed
+"Sender [Meira]
+Subject: Congratulations
+
+ 《Body》==========================================
+As Professor Yui has invented a device that allows for another method
+of communication, I shall send you a message from here.
+
+First of all, congratulations on completing the story. Regardless
+of the outcome, he escaped that city, and the assistance you've
+rendered from your world is now over. However, I'm sure you'll
+be able to appreciate on some level that this does not appear to
+be the conclusion we had hoped for.
+
```

Comment:
Let's rework this a bit, I think some things are phrased differently (correct me if I'm wrong):
```
First of all, congratulations on completing this story. Regardless
of what happened, he escaped from the city, and the 'assistance'
you can give in this world is now over.

However...

I'm sure you'll be able to understand, at least a bit, that this
is not the conclusion that we had hoped for.
```

Reply:
Happy to change it, but the only reason why I phrased it that way is because she uses a particular odd phrasing herself, and I don't know whether or not it has any significance. 

我々の望んだ結末とは程遠い = "Far from the outcome we had hoped"
The どうやら～ようです wrapped around it is a set grammatical construction that means "Appears/Seems to be", or "Is apparently".